vfs/help.go

```diff
@@ -258,6 +258,7 @@ read of the modification time takes a transaction.
     --no-modtime      Don't read/write the modification time (can speed things up).
     --no-seek         Don't allow seeking in files.
     --read-only       Only allow read-only access.
+    --links           Translate symlinks to/from regular files with a '.rclonelink' extension.
```

Comment:
There is already a local backend flag called `--links`. I think this might need to be `--vfs-links`.

Reply:
Closing, will focus on the next comment instead.